unit_tests/test_api.py

```diff
@@ -171,6 +171,16 @@ def test_reload(self):
         self.fakes.jenkins.responses[urljoin(self.api.url, "reload")] = error
         self.api.reload()
 
+    def test_restart(self):
+        """
+        The reload method POSTs a request to the '/reload' URL, expecting
+        a 503 on the homepage (which happens after redirection).
+        """
```

Comment:
Kind of annoying that success gives a failure code, making it hard to know if this was a successful failure or a genuine failure :-/